.github/CODEOWNERS

```diff
@@ -98,9 +98,6 @@
 # AzureSdkOwners:                                    @alzimmermsft
 # ServiceOwners:                                     @shenmuxiaosen @avanigupta
 
-# ServiceLabel: %Application Insights
-# ServiceOwners:                                     @azmonapplicationinsights
```

Comment:
New owner is regexrowboat

Reply:
I guess so.  @regexrowboat is from the AppInsights team.  She's adding Monitor and AppInsights tools to the azure-mcp repo currently.  I think she would be a good service team contact to list, when we need to reach out.